tests/pom.xml

```diff
@@ -10,6 +10,34 @@
 	</parent>
 	<artifactId>tests</artifactId>
 	<packaging>pom</packaging>
+
+	<build>
+		<plugins>
```

Comment:
Is absolutely working fine. Maybe These two plug-ins could be included in the parent.pom of palladiosimulator.org in the future.